### ISSUE TITLE
Forum returns "Internal Server Error" (500)

### ISSUE THREAD

**Issue by panther2** [triplea-game/triplea#9521]
This is the second day that the forum is unavailable.

Any insights, @RoiEXLab @DanVanAtta ?

**Comment by RoiEXLab** [comment]
@panther2 I'm not at home at the moment but I will have a look in a couple of hours

**Comment by RoiEXLab** [comment]
Ok, was a stupid issue. The system ran out of disk space so the database crashed.
I was able to get some temporary memory (by clearing some cache and some logs, but that won't hold too long).

@panther2 You have admin rights on the forum right? If you go to https://forums.triplea-game.org/admin/manage/uploads there is a list of all uploaded files, I tried deleting a couple of the larger orphaned files but if you have some time to spare you can remove some of the files to buy us some time to figure out a long term solution

**Comment by RoiEXLab** [comment]
Okay I was able to find a solution that should work for the forseeable future.
I added another volume to the server (2 USD a month additional cost, I hope that's ok for you @DanVanAtta ), mounted it moved all nodebb related files to this new 20GB drive, and created a symlink so the config doesn't need updating.
Now we have plenty of space again and it should take some time until the space is eaten up again.
